### ISSUE TITLE
The hyper-parameters for reproducing the XNLI results in your paper?

### ISSUE THREAD

**Issue by howardchenhd** [facebookresearch/XLM#235]
Hi, @aconneau 

I'm trying to reproduce the XNLI fine-tuning results in your paper. Setting 
"--optimizer_e adam,lr=0.000025 --optimizer_p adam,lr=0.000025" as in the doc seems doesn't work, the accuracy doesn't change. I also try to set "--optimizer_e adam,lr=0.000005 --optimizer_p adam,lr=0.000025", but the accuracy still ~2 point behind your paper. Could you tell me the exact hyper-parameters you used in your paper?

Thanks.
